packages/adblocker/src/lists.ts

```diff
@@ -251,40 +322,171 @@ export function generateDiff(
   // Set config to `debug` so that we keep track of raw lines for each filter
   const debugConfig = new Config(Object.assign({}, config, { debug: true }));
 
-  const prevRevisionFilters = getFilters(prevRevision, debugConfig);
-  const prevRevisionIds = new Set(prevRevisionFilters.map((filter) => filter.getId()));
+  const prevRevisionData = getFilters(prevRevision, debugConfig);
+  const prevRevisionIds = new Set(prevRevisionData.filters.map((filter) => filter.getId()));
 
-  const newRevisionFilters = getFilters(newRevision, debugConfig);
-  const newRevisionIds = new Set(newRevisionFilters.map((filter) => filter.getId()));
+  const newRevisionData = getFilters(newRevision, debugConfig);
+  const newRevisionIds = new Set(newRevisionData.filters.map((filter) => filter.getId()));
 
   // Check which filters were added, based on ID
-  const added: Set<string> = new Set();
-  for (const filter of newRevisionFilters) {
+  const added: Set<NetworkFilter | CosmeticFilter> = new Set();
+  for (const filter of newRevisionData.filters) {
     if (!prevRevisionIds.has(filter.getId())) {
-      added.add(filter.rawLine as string);
+      added.add(filter);
     }
   }
 
   // Check which filters were removed, based on ID
-  const removed: Set<string> = new Set();
-  for (const filter of prevRevisionFilters) {
+  const removed: Set<NetworkFilter | CosmeticFilter> = new Set();
+  for (const filter of prevRevisionData.filters) {
     if (!newRevisionIds.has(filter.getId())) {
-      removed.add(filter.rawLine as string);
+      removed.add(filter);
+    }
+  }
```

Comment:
Are these changes intended?

Reply:
it was intended but thanks to your comment I found another way to provide the same without introducing changes here